muselog/datadog.py

```diff
@@ -78,25 +78,44 @@ def default(self, obj):
 class DatadogJSONFormatter(json_log_formatter.JSONFormatter):
 
     def format(self, record):
-        message = record.getMessage()
-        extra = self.extra_from_record(record)
```

Comment:
This seemed important. Are you certain backwards compatibility is maintained?

Reply:
This is related to version 0.1.0, the original implementation was to return the extra only. The modification I made to this before, was to include the record object along with the extra object, but that contains duplicate fields as the extra object contain data that I attempted to remove from the record_dict, so instead of performing the operation twice on both objects, and still get the duplicate fields as we saw from the Datadog logs, the approach here is to customize it to our specific need. That way we do not get duplicate logs getting into Datadog.